Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -46,25 +46,45 @@ PhysicalParticleContainer::PhysicalParticleContainer (AmrCore* amr_core, int isp
         "version of Redistribute in AMReX does not work with runtime parameters");
 #endif
 
+
+#ifdef WARPX_QED
+    //Add real component if QED is enabled
+    pp.query("do_qed", do_qed);
+    if(do_qed)
+        AddRealComp("tau");
+
+    //IF do_qed is enabled, find out if Quantum Synchrotron process is enabled
+    if(do_qed)
+        pp.query("do_qed_quantum_sync", do_qed_quantum_sync);
+
+    //TODO: SHOULD CHECK IF SPECIES IS EITHER ELECTRONS OR POSITRONS!!
+#endif
+
+    //variable to set plot_flags size
```

Comment:
I've changed a little bit how plot_flag_size is calculated. I think that the code is a bit cleaner then before...

Reply:
ok

Reply:
done. It's this PR: lucafedeli88-QED-update-doc

Reply:
As discussed in PR https://github.com/ECP-WarpX/WarpX/pull/357, let's just always dump this variable if QED is on.